mailer.rst

```diff
@@ -1021,6 +1021,39 @@ you have a transport called ``async``, you can route the message there:
 Thanks to this, instead of being delivered immediately, messages will be sent to
 the transport to be handled later (see :ref:`messenger-worker`).
 
+Events
+------
+
+MessageEvent
+~~~~~~~~~~~~
+
+The MessageEvent allows the transformation of a Message and the Envelope before the email is sent::
+
+    use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+    use Symfony\Component\Mailer\Event\MessageEvent;
+    use Symfony\Component\Mime\Email;
+
+    class MailerListener implements EventSubscriberInterface
```

Comment:
```suggestion
    class MailerSubscriber implements EventSubscriberInterface
```
It would be more interesting here to change `MailerListener` to `MailerSubscriber` because the example implements the interface `EventSubscriberInterface`